src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -200,6 +207,18 @@ <h1 style="color:blue;" id = "score">Score: 0</h1>
 game = data;
 });
 
+function gameType(){
+
+    var x = document.getElementById("gameType").value;
+    console.log(x);
+    if(x == "Spanish"){
+        console.log("clicked spanish!!");
+    }
+    else{
+    console.log("clicked Normal!!");
+    $("#newGame").click();
+    }
+}
```

Comment:
I think all you have to do to make this functional is create a new route calling the other deck type, then just call that route from the button.